cdap-notifications/src/main/java/co/cask/cdap/notifications/feeds/service/MDSNotificationFeedStore.java

```diff
@@ -82,7 +82,7 @@ public NotificationFeedMds get() {
 
             return new NotificationFeedMds(new MetadataStoreDataset(mdsTable));
           } catch (Exception e) {
-            LOG.error("Failed to access app.meta table", e);
+            LOG.debug("Failed to access app.meta table: {}", e);
```

Comment:
If you want the stack trace in debug mode, you probably don't need the ``{}`` right?